Library/Homebrew/cmd/test-bot.rb

```diff
@@ -981,4 +983,35 @@ def test_bot
 
     Homebrew.failed = any_errors
   end
+
+  def sanitize_output_for_xml(output)
+    unless output.empty?
```

Comment:
You could make that `return "" if output.empty?` and reduce indentation in the remainder of the method.